docs/_Research-Data-Management/03-md.md

```diff
@@ -4,14 +4,49 @@ category: Research-Data-Management
 layout: default
 docs_css: markdown
 redirect_from: /Research-Data-Management
-empty: true
-hide: true
+empty: false
+hide: false
 ---
 
 # Metadata
 
+## Metadata is data about data
```

Comment:
```suggestion
## Metadata describes your data
```
or ## Metadata is information about your data 
personally I like the heading but maybe to much data